tests/voq/files/CRC_ERROR_SUP

```diff
@@ -0,0 +1,4 @@
+"log off; rm /tmp/esmode; log file=/tmp/esmode quiet=yes"
```

Comment:
Can we avoid having brcm code here ? is this possible to achieve this via SAI APIs ?